cli/internal/chrometracing/chrometracing_close.go

```diff
@@ -15,6 +15,10 @@ func Close() error {
 	if _, err := trace.file.Write([]byte{']'}); err != nil {
 		return err
 	}
+	// Force the filesystem to write to disk
+	if err := trace.file.Sync(); err != nil {
+		return err
+	}
```

Comment:
Just this change isn't enough to cause the `e2e.ts` test to pass?

Reply:
It seems that it's enough to make it pass, but due to the failure only appearing on Windows CI sometimes I'm not 100% sure. I moved the test so if there's another part to this failure we reduce the opportunity to fail from 5 times per run to 1 time per run.